srtcore/cache.h

```diff
@@ -44,7 +44,7 @@ written by
 #include <list>
 #include <vector>
 
-#include "common.h"
+#include "sync.h"
```

Comment:
Actually this header was included only to get access to `CGuard`, so this is better.